lib/ui/views/app_selector/app_selector_view.dart

```diff
@@ -92,7 +92,10 @@ class _AppSelectorViewState extends State<AppSelectorView> {
                       ? const AppSkeletonLoader()
                       : Padding(
                           padding: const EdgeInsets.symmetric(horizontal: 12.0)
-                              .copyWith(bottom: 80),
+                              .copyWith(
+                            bottom:
+                                MediaQuery.of(context).viewPadding.bottom + 8.0,
+                          ),
```

Comment:
is it normal for the closing parenthesis to be inequal in indentation to the opening one?

Reply:
Yes, it's normal.